### PR TITLE
update the translation credits since the latest release

### PR DESCRIPTION
**Pull request by hboetes**
https://hosted.weblate.org/projects/nicotine-plus/nicotine-plus/#reports

Range 05-03-2023/14-11-2023, anyone with more than 50 lines changed gets a mention.
Except for some credits by me. Trust me, my Slovak is non-existant.

### CODE REVIEW

File: TRANSLATORS.md [hunk 1]
```diff
@@ -76,11 +78,15 @@
  - Allan Nordhøy (comradekingu) (2021)
 
 ## Polish
+ - Mariusz (2023)
```
Comment (mathiascode) [inline TRANSLATORS.md:81]: A.k.a. mariachini. Please merge with the entry below.

File: TRANSLATORS.md [hunk 2]
```diff
@@ -93,6 +99,7 @@
  - xslendix (2023)
 
 ## Russian
+ - Kirill Feoktistov (2023)
```
Comment (mathiascode) [inline TRANSLATORS.md:102]: A.k.a. SnIPeRSnIPeR. Please merge with the entry below.

**Comment by mathiascode** [comment]
> anyone with more than 50 lines changed gets a mention

Should we lower this to, say, 20-25 lines? We don't want to clutter the credits with 1-line changes, but it's quite hard to reach 50 if you're just improving existing translations.

**Comment by hboetes** [comment]
> > anyone with more than 50 lines changed gets a mention
> 
> Should we lower this to, say, 20-25 lines? We don't want to clutter the credits with 1-line changes, but it's quite hard to reach 50 if you're just improving existing translations.

We have to put the line somewhere. My idea was you'd have to put at least one hour of work into it, over the period of a release.

**Pull request merged**